RELEASENOTES.md

```diff
@@ -1,4 +1,4 @@
 * Improve error message when `migrate-repo` is used with a target personal access token (PAT) with insufficient permissions
 * Ensure `--no-ssl-verify` flag is honored when downloading archives from GHES
-- `--bbs-project` and `--bbs-repo` are now both required in `gh bbs2gh migrate-repo` command when `--bbs-server-url` is set
-
+* `--bbs-project` and `--bbs-repo` are now both required in `gh bbs2gh migrate-repo` command when `--bbs-server-url` is set
+* Continue to next mannequin mapping in `gh gei reclaim-mannequin --csv` if a username doesn't exist
```

Comment:
Is there a reason you prefer start over dash? I guess the consensus is to use dash over star.

Reply:
I'll change them all to dash. I don't mind either way, I just wanted consistency!